protocol-specs/generators/multisignature/index.js

```diff
@@ -0,0 +1,221 @@
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	hexToBuffer,
+	intToBuffer,
+	hash,
+	getFirstEightBytesReversed,
+	bufferToIntAsString,
+	signData,
+} = require('@liskhq/lisk-cryptography');
+const BaseGenerator = require('../base_generator');
+
+const accounts = [
+	{
+		passphrase:
+			'wear protect skill sentence lift enter wild sting lottery power floor neglect',
+		privateKey:
+			'8f41ff1e75c4f0f8a71bae4952266928d0e91660fc513566ac694fed61157497efaf1d977897cb60d7db9d30e8fd668dee070ac0db1fb8d184c06152a8b75f8d',
+		publicKey:
+			'efaf1d977897cb60d7db9d30e8fd668dee070ac0db1fb8d184c06152a8b75f8d',
+		address: '2129300327344985743L',
+	},
+	{
+		passphrase:
+			'inherit moon normal relief spring bargain hobby join baby flash fog blood',
+		privateKey:
+			'de4a28610239ceac2ec3f592e36a2ead8ed4ac93cb16aa0d996ab6bb0249da2c0b211fce4b615083701cb8a8c99407e464b2f9aa4f367095322de1b77e5fcfbe',
+		publicKey:
+			'0b211fce4b615083701cb8a8c99407e464b2f9aa4f367095322de1b77e5fcfbe',
+		address: '18141291412139607230L',
+	},
+	{
+		passphrase:
+			'trim elegant oven term access apple obtain error grain excite lawn neck',
+		privateKey:
+			'8a138c0dd8efe597c8b9c519af69e9821bd1e769cf0fb3490e22209e9cabfb8df1b9f4ee71b5d5857d3b346d441ca967f27870ebee88569db364fd13e28adba3',
+		publicKey:
+			'f1b9f4ee71b5d5857d3b346d441ca967f27870ebee88569db364fd13e28adba3',
+		address: '10481548956627905381L',
+	},
+	{
+		passphrase:
+			'desk deposit crumble farm tip cluster goose exotic dignity flee bring traffic',
+		privateKey:
+			'ddc8e19d6697d6e5c1dacf6576a7169752810999918212afe14d3978b354f8aa4a67646a446313db964c39370359845c52fce9225a3929770ef41448c258fd39',
+		publicKey:
+			'4a67646a446313db964c39370359845c52fce9225a3929770ef41448c258fd39',
+		address: '3372320078773139180L',
+	},
+	{
+		passphrase:
+			'sugar object slender confirm clock peanut auto spice carbon knife increase estate',
+		privateKey:
+			'69aa94ea7ade3b7b08e277b18c1a590b2306ce5973ae8462b0b85122b180e89c57df5c3811961939f8dcfa858c6eaefebfaa4de942f7e703bf88127e0ee9cca4',
+		publicKey:
+			'57df5c3811961939f8dcfa858c6eaefebfaa4de942f7e703bf88127e0ee9cca4',
+		address: '7745870967079479156L',
+	},
+	{
+		passphrase:
+			'faculty inspire crouch quit sorry vague hard ski scrap jaguar garment limb',
+		privateKey:
+			'ffed38380998a90a2af9501f10182bc2a07922448ab383575b1e34aeddfa5482fa406b6952d377f0278920e3eb8da919e4cf5c68b02eeba5d8b3334fdc0369b6',
+		publicKey:
+			'fa406b6952d377f0278920e3eb8da919e4cf5c68b02eeba5d8b3334fdc0369b6',
+		address: '7086965981385941478L',
+	},
+];
+
+const sortKeysDescending = publicKeys =>
+	publicKeys.sort((publicKeyA, publicKeyB) => {
+		if (publicKeyA < publicKeyB) return 1;
```

Comment:
I think sorting function needs to be done by converting to the big-ending encoded integer
> For two public keys pk1 and pk2, we say that pk1 is larger than pk2, and write pk1 > pk2, if and only if the 256-bit representation of pk1 interpreted as a big-endian encoded integer is larger than the one of pk2.

Reply:
Changed to compare using BigInts. Checked with research and the endiandness should not matter in this case as the objective is to have the keys sorted. This LIP is gonna be updated a bit so I think this is going to be one of the corrections.